engine/src/main/resources/org/camunda/bpm/engine/db/drop/activiti.cockroachdb.drop.history.sql

```diff
@@ -23,6 +23,7 @@ drop index ACT_IDX_HI_PRO_INST_PROC_TIME;
 drop index ACT_IDX_HI_PI_PDEFID_END_TIME;
 drop index ACT_IDX_HI_PRO_INST_ROOT_PI;
 drop index ACT_IDX_HI_PRO_INST_RM_TIME;
+drop index ACT_IDX_HI_PRO_RST_PRO_INST_ID;
```

Comment:
See [point A](https://github.com/camunda/camunda-bpm-platform/pull/4420#issuecomment-2337712471) in the ticket thread